<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>241</width>
    <height>391</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>J's Calculator</string>
  </property>
  <property name="documentMode">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="Display">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>241</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>27</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
  qproperty-alignment: 'AlignVCenter | AlignRight';
  border: 1px solid gray;
  background-color : rgb(30, 30, 30);
  color: rgb(250, 250, 250);
}

background-color : rgb(30, 30, 30);</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonClear">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(215, 215, 215);
  border: 1px solid gray;
}</string>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonPlusMinus">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>90</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(215, 215, 215);
  border: 1px solid gray;
}</string>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonDivide">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>90</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(255, 151, 57);
  color: white; 
  border: 1px solid gray;
}</string>
    </property>
    <property name="text">
     <string>รท</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonPercent">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>90</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(215, 215, 215);
  border: 1px solid gray;
}</string>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>150</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid gray;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button8">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>150</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid gray;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonMultiply">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>150</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(255, 151, 57);
  color: white; 
  border: 1px solid gray;
}</string>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button9">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>150</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid gray;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>210</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid gray;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button5">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>210</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid gray;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonSubtract">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>210</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(255, 151, 57);
  color: white; 
  border: 1px solid gray;
}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button6">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>210</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid gray;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>270</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid gray;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>270</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid gray;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonAdd">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>270</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(255, 151, 57);
  color: white; 
  border: 1px solid gray;
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>270</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid gray;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button0">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>330</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid gray;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonEquals">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>330</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(255, 151, 57);
  color: white; 
  border: 1px solid gray;
}</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonDot">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>330</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(215, 215, 215);
  border: 1px solid gray;
}</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
